components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.test.tsx

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper';
+
+import EnterpriseEditionLeftPanel, {EnterpriseEditionProps} from './enterprise_edition_left_panel';
+
```

Comment:
please, remove extra blank line. Before pushing changes, if you run `make fix-style` it will do it for you or will warn you about what needs to be fixed.

Reply:
Fixed!